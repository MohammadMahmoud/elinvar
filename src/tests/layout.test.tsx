import React from 'react';
import renderer from 'react-test-renderer';
import Layout from '../components/home';

test('snapshot layout', () => {
  const layoutTree = renderer.create(<Layout />).toJSON();
  expect(layoutTree).toMatchSnapshot();
});
