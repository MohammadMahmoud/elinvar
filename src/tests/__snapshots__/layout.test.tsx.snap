// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <div>
      <h1>
        Any place in your app!
      </h1>
      <form
        action="#"
        onReset={[Function]}
        onSubmit={[Function]}
      >
        <input
          name="email"
          onBlur={[Function]}
          onChange={[Function]}
          type="email"
          value=""
        />
        <input
          name="password"
          onBlur={[Function]}
          onChange={[Function]}
          type="password"
          value=""
        />
        <button
          disabled={false}
          type="submit"
        >
          Submit
        </button>
      </form>
    </div>
  </header>
</div>
`;
